apps/src/lib/node/ledger/storage/rocksdb.rs

```diff
@@ -775,6 +779,35 @@ impl DB for RocksDB {
                 None => unknown_key_error(path)?,
             }
         }
+        // Restore subtrees of Merkle tree
+        if let Some(epoch) = epoch {
+            for st in StoreType::iter_subtrees() {
+                let key_prefix = subtree_key_prefix(st, epoch);
+                let root_key = key_prefix
+                    .push(&"root".to_owned())
+                    .map_err(Error::KeyError)?;
+                if let Some(bytes) = self
+                    .0
+                    .get_cf(block_cf, &root_key.to_string())
+                    .map_err(|e| Error::DBError(e.into_string()))?
+                {
+                    merkle_tree_stores.set_root(
+                        st,
+                        types::decode(bytes).map_err(Error::CodingError)?,
+                    );
+                }
+                let store_key = key_prefix
+                    .push(&"store".to_owned())
+                    .map_err(Error::KeyError)?;
```

Comment:
```suggestion
                let store_key = key_prefix
                    .with_segment("store".to_owned());
```
this method doesn't need to error out